src/instructlab/eval/mt_bench_common.py

```diff
@@ -250,9 +252,10 @@ def play_a_match_single(
 def chat_completion_openai(
     openai_client, model, conv, temperature, max_tokens, merge_system_user_message=False
 ) -> str:
-    output = API_ERROR_OUTPUT
```

Comment:
It's unclear whether it's appropriate to remove this condition vs allow it to be included in the error rate.  While I prefer this strict interpretation, it would potentially create a lot of flakiness in the results.  So it's a tradeoff of completeness/accuracy vs flakiness.